release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -1156,6 +1156,29 @@ The Thanos Querier aggregates and optionally deduplicates core {product-title} m
 
 The Virtual Machine `Pending Changes` alert is now more informative. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1862801[*BZ#1862801*])
 
+[id="ocp-4-6-insights-operator"]
+=== Insights Operator
+
+[id="ocp-4-6-insights-operator-data-collection-enhancements”]
```

Comment:
@pneedle-rh Not sure what is throwing off the build here. The only thing I noticed was the closing quotations in the ID don't match. That last one appears to be a special char version copied probably from a Google Doc. I would try replacing that and seeing if that fixes it (if you, too, don't know what's causing the issue).

Reply:
@codyhoag thank you for your review and for your guidance on this. I have just pushed an update after replacing that character and will monitor the progress of the new build.

Reply:
The subsequent build has succeeded. Thanks again!